unsigned-security-token.cs

```diff
@@ -0,0 +1,10 @@
+services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
+            {
+
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    RequireSignedTokens = false,
```

Comment:
the testing mechanism expects you to have a `ruleid: unsigned-security-token` comment above this line!